docs/fsharp-core-notes.md

```diff
@@ -1,126 +1,124 @@
----
-layout: default
-title: Notes on FSharp.Core
-subtitle: This technical guide discusses the FSharp.Core library.  
----
-
 # Notes and Guidance on FSharp.Core
-{:.no_toc}
 
-This technical guide discusses the FSharp.Core library.  Please help improve this guide by editing it and submitting a pull-request.
+This technical guide discusses the FSharp.Core library.
+
+Reference documentation for FSharp.Core can be found here: https://fsharp.github.io/fsharp-core-docs/
 
 Much of the guidance below applies to any .NET library respecting binary compatibility.
 
-### FSharp.Core is binary compatible
+## FSharp.Core is binary compatible
 
-FSharp.Core is binary compatible across versions of the F# language. For example, FSharp.Core `5.0.0.0` (F# 5.0) is binary compatible with
-`4.7.0.0` (F# 4.7), `4.6.0.0` (F# 3.6), `4.4.0.0` (F# 4.0) , `4.4.1.0` (F# 4.1), , `4.4.3.0` (F# 4.1+)  and so on.
+FSharp.Core is binary compatible across versions of the F# language. For example, this means you can create a newer project with a newer FSharp.Core in an older codebase and things should generally "just work".
 
-Likewise, FSharp.Core is binary compatible from "netstandard" profiles to actual runtime implementations.
-For example, FSharp.Core  for `netstandard2.0` is binary compatible with the runtime implementation assembly `4.7.0.0` and so on.
+**Binary compatibility means that a component built for X can bind to Y at runtime. It doesn't mean that Y behaves 100% the same as X, though.** For example, an older compiler that doesn't know how to understand `inref<'T>` referencing a newer FSharp.Core that has `inref<'T>` defined may not behave correctly if `inref<'T>` is used in source.
 
-Binary compatibility means that a component built for X can instead bind to Y at runtime.
-It doesn't mean that Y behaves 100% the same as X (some bug fixes may have been made, and Y may have more functionality than X).
+### Note on legacy FSharp.Core binaries - 2.0.0 and below
 
-### Application v. Library v. Script
-{:.no_toc}
+FSharp.Core 4.0.0.0 and upwards is not binary compatible with apps built with F# 2.0 and FSharp.Core 2.0.0.0. Some types such as `Tuple` and `Bigint` moved to be supported in .NET. At the time (.NET Framework 2.0-3.5) had less support for things and F#, like most of .NET, had to make a breaking change to support .NET Framework 4.0 and higher.
```

Comment:
My understanding is that FSharp.Core was always binary compatible from 2.0.0.0 onewards - type forwarders were/are used.  Anyway i don't think we need this point since older versions are now very rare.

Reply:
@dsyme, you are correct, we do:

![image](https://user-images.githubusercontent.com/5175830/109211538-38012980-7763-11eb-841f-30585163c9d4.png)

Reply:
I'll just remove it like before